crates/driver/solvers.example.yaml

```diff
@@ -0,0 +1,10 @@
+solvers:
```

Comment:
If we have an example yaml config then we might as well test it with the function that is supposed to load it to make sure the example and the code don't go out of sync.